### ISSUE TITLE
Latest version has broken node-gyp

### ISSUE THREAD

**Issue by MarshallOfSound** [npm/npmlog#55]
As of 2 hours ago new installs of `electron-rebuild` started failing with this error

```
gyp ERR! UNCAUGHT EXCEPTION 
gyp ERR! stack TypeError: Property description must be an object: Error: not found: python2
gyp ERR! stack     at getNotFoundError (/Users/sattard/projects/electron-rebuild/node_modules/which/which.js:13:12)
gyp ERR! stack     at F (/Users/sattard/projects/electron-rebuild/node_modules/which/which.js:68:19)
gyp ERR! stack     at E (/Users/sattard/projects/electron-rebuild/node_modules/which/which.js:80:29)
gyp ERR! stack     at /Users/sattard/projects/electron-rebuild/node_modules/which/which.js:89:16
gyp ERR! stack     at /Users/sattard/projects/electron-rebuild/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /Users/sattard/projects/electron-rebuild/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:123:15)
gyp ERR! stack     at Function.defineProperty (native)
gyp ERR! stack     at EventEmitter.<anonymous> (/Users/sattard/projects/electron-rebuild/node_modules/npmlog/log.js:185:14)
gyp ERR! stack     at EventEmitter.<anonymous> (/Users/sattard/projects/electron-rebuild/node_modules/npmlog/log.js:281:23)
gyp ERR! stack     at Object.<anonymous> (/Users/sattard/projects/electron-rebuild/node_modules/node-gyp/lib/configure.js:389:18)
gyp ERR! stack     at F (/Users/sattard/projects/electron-rebuild/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/Users/sattard/projects/electron-rebuild/node_modules/which/which.js:80:29)
gyp ERR! stack     at /Users/sattard/projects/electron-rebuild/node_modules/which/which.js:89:16
gyp ERR! stack     at /Users/sattard/projects/electron-rebuild/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /Users/sattard/projects/electron-rebuild/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:123:15)
gyp ERR! System Darwin 16.5.0
```

This error was tracked back to this commit --> https://github.com/npm/npmlog/commit/c07471288ebe3ea81b0d75e43c7edd584d85b8cf

Which appears to be invalid syntax as the third parameter for `Object.defineProperty` should be an object (the descriptor).  See [the docs](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty)

This needs to be reverted ASAP as all new installs of `node-gyp` will be failing.

/cc @iarna

**Comment by MarshallOfSound** [comment]
A workaround for anyone experiencing issues as a result of this is to pin `npmlog` in your `package.json` to version `4.1.0`.

**Comment by IRun26Point2** [comment]
we are also seeing this broken during jshint tests:

```
2017-06-26T22:49:20.8819055Z E:\agent_work\91\s\node_modules\npmlog\log.js:185
2017-06-26T22:49:20.8819055Z       Object.defineProperty(arg, 'stack', arg.stack = stack = arg.stack + '')
2017-06-26T22:49:20.8819055Z              ^
2017-06-26T22:49:20.8819055Z 
2017-06-26T22:49:20.8819055Z TypeError: Property description must be an object: Error: spawn phantomjs ENOENT
2017-06-26T22:49:20.8819055Z     at notFoundError (E:\agent_work\91\s\node_modules\cross-spawn\lib\enoent.js:11:11)
2017-06-26T22:49:20.8819055Z     at verifyENOENT (E:\agent_work\91\s\node_modules\cross-spawn\lib\enoent.js:46:16)
2017-06-26T22:49:20.8819055Z     at ChildProcess.cp.emit (E:\agent_work\91\s\node_modules\cross-spawn\lib\enoent.js:33:19)
2017-06-26T22:49:20.8819055Z     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
2017-06-26T22:49:20.8819055Z     at Function.defineProperty (native)
2017-06-26T22:49:20.8819055Z     at EventEmitter.<anonymous> (E:\agent_work\91\s\node_modules\npmlog\log.js:185:14)
2017-06-26T22:49:20.8819055Z     at EventEmitter.<anonymous> (E:\agent_work\91\s\node_modules\npmlog\log.js:281:23)
2017-06-26T22:49:20.8819055Z     at Process.onError (E:\agent_work\91\s\node_modules\testem\lib\utils\process.js:48:7)
2017-06-26T22:49:20.8819055Z     at emitOne (events.js:101:20)
2017-06-26T22:49:20.8819055Z     at ChildProcess.emit (events.js:188:7)
2017-06-26T22:49:20.8819055Z     at ChildProcess.cp.emit (E:\agent_work\91\s\node_modules\cross-spawn\lib\enoent.js:36:37)
2017-06-26T22:49:20.8819055Z     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
```

**Comment by iarna** [comment]
This is fixed in npmlog@4.1.2
